docs/tutorials/flow-app-quickstart.mdx

```diff
@@ -2,43 +2,43 @@
 
 ---
 
-**Last Updated:** January 11th 2022
+**Last Updated:** June 15th 2022
 
 > **Note**: This page will walk you through a very bare bones project to get started building a web3 dapp using the Flow Client Library (FCL). If you are looking for a clonable repo, Flow community members have created quickstart templates for different JavaScript frameworks (e.g. [Next.js](https://github.com/muttoni/fcl-nextjs-quickstart), [SvelteKit](https://github.com/muttoni/fcl-sveltekit-quickstart), [Nuxt](https://github.com/bluesign/nuxt3-fcl)). You can consult the complete list [here](https://github.com/ph0ph0/Get-The-Flow-Down#fcl).
 
-## Video Walkthrough
-
-Below is a video walkthrough covering the contents of this tutorial. You are welcome to watch or skip the video walkthrough depending on your style of learning.
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/uEoh9SnjqCk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ## Introduction
 
-FCL-JS is the easiest way to start building decentralized applications. FCL (aka Flow Client Library) wraps much of the logic you'd have to write yourself on other blockchains. Follow this quick start and you'll have a solid overview of how to build a shippable dapp on Flow.
+FCL-JS (Flow Client Library) is the easiest way to build decentralized applications. FCL wraps much of the logic you would have to write on other blockchains. Follow this quick start, and you will have a solid overview of building a shippable dapp on Flow.
 
-We're going to make an assumption that you know or understand React; however, the concepts should be easy to understand and transfer to another framework. While this tutorial will make use of Cadence (Flow's smart contract language), you do not need to know it. Instead, we recommend later diving into [learning the Cadence language](https://docs.onflow.org/cadence/) once you've gotten the core FCL concepts down.
-
-In this tutorial, we are going to interact with an existing smart contract on Flow's testnet known as the [Profile Contract](https://testnet.flowscan.org/contract/A.ba1132bc08f82fe2.Profile). Using this contract, we will create a new profile and edit the profile information, both via a wallet. In order to do this, the FCL concepts we'll cover are:
+## Prerequisite
+- Basic understanding of React
+- Basic understanding of Cadence programming language 
+- Basic understanding of CLI knowledge
```

Comment:
> I'd remove that; devs would need to know how CLIs work. And we cannot expect them to know the Flow CLI prior to running the quickstart, right?

I think it's necessary to let the readers know they need CLI experience before going through the quick start.

Reply:
this is something that goes back to our target audience. I'll share our personas in a DM with you but the tldr is we are targeting seasoned web2 developers. think of devs with 3+ years of full-time experience. The argument from that understanding is that experienced developers will know how to interact with terminals/CLIs, so it should be assumed (just like knowing Git and having that installed on your machine). What is more important here, because we're asking them to use Node, is having the right version installed. I suggest we add that